From: Dinny Wu <dinny.wu@gmail.com>
Date: Thu, 26 Jul 2012 14:12:51 +0000
Subject: [PATCH] Fix the format warning when building applets/usage_pod.c
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

When compiling busybox with gcc 4.6.3, it shows below warning:

applets/usage_pod.c: In function ‘main’:
applets/usage_pod.c:74:3: warning: format not a string literal and no format arguments [-Wformat-security]

Signed-off-by: Dinny Wu <dinny.wu@gmail.com>
---
 applets/usage_pod.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/applets/usage_pod.c b/applets/usage_pod.c
index 0b1c4aadb8c4..a67e8b4ac348 100644
--- a/applets/usage_pod.c
+++ b/applets/usage_pod.c
@@ -71,7 +71,7 @@ int main(void)
 		} else {
 			printf(", ");
 		}
-		printf(usage_array[i].aname);
+		printf("%s", usage_array[i].aname);
 		col += len2;
 	}
 	printf("\n\n");
